src/test/rpc/Tx_test.cpp

```diff
@@ -528,6 +546,26 @@ class Tx_test : public beast::unit_test::suite
                 client.status.ok(), client.reply);
         };
 
+        auto grpcAccountTx = [&grpcPort](auto hash, auto binary, auto account) {
```

Comment:
I don't think this needs to be a template, I'd replace the `auto`s with their actual parameter types. I'd also pass the first and third parameter by `const&`

Reply:
fixed in `d10a4a02`